Documentation/operations/performance/tuning.rst

```diff
@@ -198,7 +198,30 @@ Running with Hubble observability enabled can come at the expense of
 performance. The overhead of Hubble is somewhere between 1-15% depending
 on your network traffic patterns and Hubble aggregation settings.
 
-In order to optimize for maximum performance, Hubble can be disabled:
+In clusters with a huge amount of network traffic, cilium-agent might spend
+a significant portion of CPU time on processing monitored events when Hubble
+is enabled. To prevent high CPU utilization in such a case, you can set limits
+on how many events can be generated by datapath code. Two limits are possible
+to configure:
+
+* Rate limit - limits how many events on average can be generated
+* Burst limit - limits the number of events that can be generated in a span of 1 second
+
+To enable eBPF Event Rate Limiting with a rate limit of 10000 and a burst limit of 50000:
+
```

Comment:
I think we should add a warning block here that when the event map rate limit is enabled, then it will impact the Hubble metrics reliability as we might drop events.